libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -816,6 +853,10 @@ export class RushConfiguration {
 
       // Consumer relationships will be established the first time one is requested
     }
+
+    if (this.hasSplitWorkspaceProject) {
```

Comment:
A small bug:  If the "hasSplitWorkspaceProject" setting is missing in rush.json, then I get:

```
$ testrush --debug update

Rush Multi-Project Build Tool 5.71.0 (unmanaged) - https://rushjs.io
Node.js version is 14.20.0 (LTS)

ERROR: Cannot read property 'hasSplitWorkspaceProject' of undefined
```